ksy_schema.json

```
@@ -263,7 +296,13 @@
                 "cases"
               ]
             }
-          ]
+          ],
+          "description": "specifies the default byte order used in all fields of type `sX` or `uX` of the defining type and all its subtypes\n\nthis field is required if you use types `sX` or `uX` without explicitly defined endianness"
```

```suggestion
          "description": "specifies the default byte order for all fields of type `sX` or `uX` in the current type and all its subtypes\n\nthis field is required if you use types `sX` or `uX` without explicitly defined endianness"
```

Also, this statement

> this field is required if you use types `sX` or `uX` without explicitly defined endianness

is not entirely true - if you use only `u1`/`s1` types, you don't need to specify any endianness (and it's actually redundant and discouraged - see e.g. https://github.com/kaitai-io/kaitai_struct_formats/pull/355#discussion_r524701315).

And instead of just "without explicitly defined endianness", you could rephrase that as _"if you use types `sX` or `uX` without an explicit `le` / `be` suffix (e.g. `u2be`)"_. We should not assume here that the reader knows how to "explicitly declare the endianness".